android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddleware.kt

```diff
@@ -45,10 +45,14 @@ class TranslationsMiddleware(
     ) {
         // Pre process actions
         when (action) {
-            is InitAction ->
+            is InitAction -> {
                 scope.launch {
                     requestEngineSupport(context)
                 }
+                scope.launch {
```

Comment:
Plan on refactoring this after #5680 lands to match new ordering accordingly. (e.g., posting after the engine support is requested and successfully true.)